client/components/data/followers-data/index.jsx

```diff
@@ -3,9 +3,8 @@
  *
```

Comment:
I smoke tested `followers-data` and `email-followers-data` and there seem to be no regressions 👍 

Interesting, first time I've seen the `DataComponent` wrapper in Calypso 🤔